tests/end-to-end/pkg/common/gqlclient.go

```diff
@@ -0,0 +1,55 @@
+package common
+
+import (
+	"crypto/tls"
+	"fmt"
+	"net/http"
+	"os"
+	"time"
+
+	gcli "github.com/machinebox/graphql"
+)
+
+func NewAuthorizedGraphQLClient(bearerToken string) *gcli.Client {
+	authorizedClient := newAuthorizedHTTPClient(bearerToken)
+	return gcli.NewClient(getDirectorGraphqlURL(), gcli.WithHTTPClient(authorizedClient))
+}
+
+func NewAuthorizedGraphQLClientWithCustomURL(bearerToken, url string) *gcli.Client {
+	authorizedClient := newAuthorizedHTTPClient(bearerToken)
+	return gcli.NewClient(url, gcli.WithHTTPClient(authorizedClient))
+}
+
+func getDirectorGraphqlURL() string {
+	url := os.Getenv("DIRECTOR_URL")
+	if url == "" {
```

Comment:
This can be refactor to something like:
```go
	if url == "" {
		url = "http://127.0.0.1:3000"
	} 
	url = url + "/graphql"
```

But please consider also using the url and path packages to work with URLs and path concatenation.